### PR TITLE
docs(confguring-labs): add info about Googles host maintenance events

### PR DESCRIPTION
**Pull request by 0x-r4bbit**

### CODE REVIEW

File: src/guide/configuring-labs.md [hunk 1]
```diff
@@ -130,6 +130,14 @@ GPU support isn't available to everyone yet. However, it can easily be enabled b
 
 <p class="tip">During private beta, **GPU support is only enabled for Patreon backers**. Thank you for your support!</p>
 
+### A note on host maintenance
+
+Since we're using Google Cloud Platform, GPU instances will terminate for host maintenance events and restart automatically. These maintenance events typically occur once per week.
+
+This obviously has an impact on long running tasks like labs that perform training. We're working on iron these things out.
```
Comment (cburgdorf) [inline src/guide/configuring-labs.md]: _iron_ -> _ironing_

>long running tasks like labs that perform training

Are the any other tasks that would have an impact on? I think we should rephrase this as

>This means that there is a slight chance that an execution may be unexpectedly terminated by the system every once in a while.

Comment (0x-r4bbit) [reply src/guide/configuring-labs.md]: Okay
